hooks/test

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -ex
+
+# Create a container with test and production postgres databases.
+docker pull postgres:9.5;
+docker run -d --name pg postgres:9.5;
+
+# Sleep to let PostgreSQL start up.
+sleep 10;
+
+# Create production and test databases.
+docker exec -u postgres pg psql -c "create user simplified with password 'test';";
+docker exec -u postgres pg psql -c "create database docker_prod;";
+docker exec -u postgres pg psql -c "create database docker_test;";
+docker exec -u postgres pg psql -c "grant all privileges on database docker_prod to simplified;";
+docker exec -u postgres pg psql -c "grant all privileges on database docker_test to simplified;";
+
+pghost=$(docker inspect pg --format="{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}")
+echo $pghost
+
+# Create a base and deployment container with this tag.
+docker run -d -p 80:80 \
+  -e SIMPLIFIED_DB_TASK="init" \
+  -e SIMPLIFIED_TEST_DATABASE="postgres://simplified:test@${pghost}:5432/docker_test" \
+  -e SIMPLIFIED_PRODUCTION_DATABASE="postgres://simplified:test@${pghost}:5432/docker_prod" \
+  --name circ --rm "$IMAGE_NAME" 
+
+# Run the tests in the container.
+docker exec -u simplified circ /bin/bash \
+  -c "cd ~/circulation && source env/bin/activate && ./verbose-test && cd core && ./test -v";
+
+# If this is a scripts container, there's nothing more to be done. 
+if [[ ${IMAGE_NAME} == *"scripts"* ]]; then exit 0; fi
```

Comment:
The later tests don't apply to circ-scripts, so now the tests will exit here when building a scripts container.